classes/03class/exercises/c03-make01/frdvo/Makefile

```diff
@@ -0,0 +1,27 @@
+MY_NAME ?= Fernando Rolnik
+TAG ?= $(shell git rev-parse --short HEAD)
+CR_LOGIN ?= frdvo
+C_NAME ?= make01
+
+.PHONY: build
+build:
+	@echo "\033[1;33mGetting the short commit sha and saving on index.html\033[0m"
```

Comment:
Tip: with `make -s` you use the silent mode, supressing all the command outputs instead of using `@` on every line